test/fixtures/rules/std_rule_cases/LT15.yml

```diff
@@ -0,0 +1,82 @@
+rule: LT15
+
+test_pass_no_empty_lines:
+  pass_str: |
+    SELECT foo
+    FROM bar
+
+test_pass_one_empty_line:
+  pass_str: |
+    SELECT foo
+
+    FROM
+      bar
+  configs:
+    rules:
+      layout.newlines:
+        maximum_empty_lines_inside_statements: 1
+
+test_fail_no_empty_lines:
+  fail_str: |
+    SELECT foo
+
+    FROM
+      bar
+  fix_str: |
+    SELECT foo
+    FROM
+      bar
+  configs:
+    rules:
+      layout.newlines:
+        maximum_empty_lines_inside_statements: 0
+
+test_fail_one_empty_line_between_statements:
+  fail_str: |
+    SELECT foo
+    FROM
+      bar
+    ;
+
+
+    SELECT foo
+    ;
+  fix_str: |
+    SELECT foo
+    FROM
+      bar
+    ;
+
+    SELECT foo
+    ;
+  configs:
+    rules:
+      layout.newlines:
+        maximum_empty_lines_between_statements: 1
+
+test_fail_bad_edge_case:
```

Comment:
@alanmcruickshank here is the edge case that i'd like to avoid

Reply:
I see what you mean 🤔 . I'm really not sure this is a problem. Personally I feel like the semicolon "belongs" to the preceding statement, but I also think a comma "belongs" to the preceding line. Leaving this open so that we're a bit more lenient on whether the "between statements" gap is _before_ or _after_ the semicolon I think is ok.

I don't think hard coding the behaviour is worth the extra complexity unless we also give people the option to configure it in case they want to do.... _leading semicolons?!_.

Reply:
I think I' trying to say: I think this PR is ok as-is. We can always optionally tighten it later.

Reply:
I'll add a comment to that test case to explain